st2actions/st2actions/notifier.py

```diff
@@ -69,15 +80,15 @@ def _post_notify_triggers(self, liveaction):
 
         if notify.on_complete:
             self._post_notify_subsection_triggers(
-                liveaction, notify.on_complete, default_message_suffix='completed.')
+                liveaction, execution_id, notify.on_complete, default_message_suffix='completed.')
```

Comment:
Yeah, I saw this code and I had the same thought. I'll add a commit. Wanted to get this reviewed first.